COMPILING

```diff
@@ -48,13 +48,14 @@ We assume that you have a Debian/Ubuntu or Red Hat-like distribution.
 
    cd cbmc-git/src
    make minisat2-download
-   make
+   make CXX=g++-6
 
    On Redhat/Fedora etc., do
 
    cd cbmc-git/src
    make minisat2-download
-   make CXX=g++-6
+   scl enable devtoolset-6 bash
```

Comment:
Yes, let's go for the "scl enable devtoolset-6 bash" change; it's better that the regressions work out-of-the box